src/xdocs/config_naming.xml

```diff
@@ -270,11 +270,18 @@
           <code>AbstractClassName</code> to checks names, but ignore
           missing <code>abstract</code> modifiers:
         </p>
+        <p>Configuration:</p>
```

Comment:
@romani We are adding this `Configuration` label to all our examples?
Some checks already say `An example of how to configure the check is:`. http://checkstyle.sourceforge.net/config_naming.html#CatchParameterName_Examples
Will we be changing the wording there too?

Reply:
for now I am not 100% sure what is right way, lets keep adding "Configuration" for now, it might be useful anchor for us in future (ones we start generating xdoc from javadoc). As you see now, having "copy paste" is pain to maintain.